src/controls/listItemAttachments/ListItemAttachments.tsx

```diff
@@ -274,8 +274,8 @@ export class ListItemAttachments extends React.Component<IListItemAttachmentsPro
                     directionalHint={DirectionalHint.rightCenter}>
 
                     <DocumentCard
-                      onClickHref={!openAttachmentsInNewWindow && `${encodeURIComponent(file.ServerRelativeUrl)}?web=1`}
-                      onClick={openAttachmentsInNewWindow && (() => window.open(`${encodeURIComponent(file.ServerRelativeUrl)}?web=1`, "_blank"))} // JJ - 20200613 - needed to support Microsoft Teams
+                      onClickHref={!openAttachmentsInNewWindow && `${(file.ServerRelativeUrl)}?web=1`} //NB - 20230607 - Fix for issue 1541
```

Comment:
@NishkalankBezawada no need to add comments as this is not a generally adopted practice within the repository so I will remove them after the merge